src/Template/Bake/Template/index.ctp

```diff
@@ -39,20 +39,21 @@ $fields = collection($fields)
     endforeach;
 %>
     </ul>
-</div>
-<div class="<%= $pluralVar %> index large-10 medium-9 columns">
+</nav>
+<div class="<%= $pluralVar %> index large-9 medium-8 columns" id="content">
```

Comment:
Doing stuff semantically is like a 90% automation to SEO optimization of HTML. In rare cases you want something to "look" like ```<h2>``` but actually be just a ```<p>```. Having a lot of class names and non-semantic wrappers (```<div>```s) make it hard to create html for layouts, and view template files as well as content that is responsive and semantic/search-engine optimized.

In my defense / pro your argument: The way I did my last huge PRs on this matter. It was one big and hard to maintain file. This doesn't have to be the case and I was not focusing on maintainability (but should have).